src/subscription_manager/managercli.py

```diff
@@ -53,6 +55,38 @@
 
 log = logging.getLogger(__name__)
 
+conf = config.Config(rhsm.config.initConfig())
+
+SM = "subscription-manager"
+
+
+class SubscribeCommand(AttachCommand):
```

Comment:
Why do you want to add back this alias? It seems unrelated to this PR.

Reply:
Wasn't sure about this so left it knowing it would probably be brought up.  :)  Removed